New Package: spotify-qt

T-Dynamos opened the pull request:
This is my first package on termux

kcubeterm commented on tur/spotify-qt:
```diff
@@ -0,0 +1,8 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/kraxarn/spotify-qt"
+TERMUX_PKG_DESCRIPTION="Lightweight Spotify client using Qt"
+TERMUX_PKG_LICENSE="GNU General Public License v3.0"
+TERMUX_PKG_MAINTAINER="@termux"
```
either put @tur or your own username.

T-Dynamos replied:
Ok

T-Dynamos commented:
I added qt5-base but it gave error 
```
ERROR: Package spotify-qt depends on non-existing package "qt5-qtbase"
```
When i removed it
```
  Could not find a package configuration file provided by "Qt5" with any of
  the following names:
    Qt5Config.cmake
    qt5-config.cmake
```

kcubeterm commented:
need to add x11-packages in repo.json  as well. I am working on it.

licy183 commented on tur/spotify-qt/build.sh:
```diff
@@ -0,0 +1,8 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/kraxarn/spotify-qt"
+TERMUX_PKG_DESCRIPTION="Lightweight Spotify client using Qt"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@Tur"
```
I'd prefer `@termux-user-repository`. Actually, `@Tur` is a github user.

licy183 commented on tur/spotify-qt/build.sh:
```diff
@@ -0,0 +1,8 @@
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@Tur"
+TERMUX_PKG_VERSION="3.8"
+TERMUX_PKG_SRCURL="https://github.com/kraxarn/spotify-qt/archive/refs/tags/v3.8.tar.gz"
```
Please replace `3.8` with `$TERMUX_PKG_VERSION`.

licy183 commented on tur/spotify-qt/build.sh:
```diff
@@ -0,0 +1,8 @@
+TERMUX_PKG_VERSION="3.8"
+TERMUX_PKG_SRCURL="https://github.com/kraxarn/spotify-qt/archive/refs/tags/v3.8.tar.gz"
+TERMUX_PKG_SHA256="deb12800b9fb4f501f8dfc97a2fdc4989849fe8bbf6f38d79503214e03afe5ad"
+TERMUX_PKG_DEPENDS="qt5-qtbase, zlib"
```
It seems to depend on `qt5-qtsvg` too.

2096779623 commented:
https://github.com/termux/termux-packages/issues
